aws_topology/stackstate_checks/aws_topology/resources/cloudformation.py

```diff
@@ -51,7 +52,7 @@ def no_arn(region=None, account_id=None, resource_id=None, **kwargs):
     'AWS::ApiGateway::Resource': create_resource_arn,
     'AWS::ApiGateway::Method': create_method_arn,
     'AWS::ElasticLoadBalancing::LoadBalancer': create_elb_arn,  # TODO odd one
-    'AWS::Redshift::Cluster': no_arn,
+    'AWS::Redshift::Cluster': create_redshift_cluster_arn,
```

Comment:
Adding it here will also mean the a reference from a cloudformation template will use this to construct arn from physical id in the template resources.

Reply:
Will make sure there is a test